collections/_article/upcoming-serious-web-performance-boost.md

```diff
@@ -0,0 +1,52 @@
+
+| | Without WASM SIMD | With WASM SIMD | Improvement (approx.) |
+| :---: | :---: | :---: | :---: |
+| Test links | <a href="https://adamscott.github.io/godot-physics-demo-without-simd/" target="_blank">Link</a> | <a href="https://adamscott.github.io/godot-physics-demo-with-simd/" target="_blank">Link</a> | - |
+| Firefox 138<br>("+100 barrels" 3 times) | ![Firefox 138 "+ 100 barrels" 3 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-firefox138-3times-without-wasm-simd.webp) | ![Firefox 138 "+ 100 barrels" 6 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-firefox138-3times-with-wasm-simd.webp) | **2×** |
+| Firefox 138<br>("+100 barrels" 6 times) | ![Firefox 138 "+ 100 barrels" 3 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-firefox138-6times-without-wasm-simd.webp) | ![Firefox 138 "+ 100 barrels" 6 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-firefox138-6times-with-wasm-simd.webp) | **10.17×**\* |
+| Chromium 134<br>("+100 barrels" 3 times) | ![Chromium 134 "+ 100 barrels" 3 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-chromium134-3times-without-wasm-simd.webp) | ![Chromium 134 "+ 100 barrels" 6 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-chromium134-3times-with-wasm-simd.webp) | **1.37×** |
+| Chromium 134<br>("+100 barrels" 6 times) | ![Chromium 134 "+ 100 barrels" 3 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-chromium134-6times-without-wasm-simd.webp) | ![Chromium 134 "+ 100 barrels" 6 times without SIMD](/storage/blog/upcoming-web-serious-performance-boost/calinou-chromium134-6times-with-wasm-simd.webp) | **14.17×**\* |
+
+_\*Please note that once the physics engine enters a "spiral of death", it is common for the framerate to drop to single digits, SIMD or not. These tests don't prove 10× to 15× CPU computing speed improvements, but rather that games will be more resilient to framerate drops on the same machine in the same circumstances. The 1.5× to 2× numbers are more representative here of the performance gains by WASM SIMD._
+
+## What it means for your games
+
+Starting with 4.5 dev 5, you can expect your Web games to run a little bit more smoothly, without having to do anything. Especially when things get chaotic (for your CPU). It isn't a silver bullet for poorly optimized games, but it will help nonetheless. And note that it cannot do anything for GPU rendering bottlenecks.
+
+Be aware that the stress tests are meant by nature to only test the worst case scenarios, so you may not see such large improvements in normal circumstances. But it's nice to see such stark improvements when the worst happens.
+
+## Availability
+
+The 4.5 release official templates will only support WebAssembly SIMD compatible browsers in order to keep the template sizes small.
```

Comment:
```suggestion
Starting from 4.5 stable, the official release templates will only support WebAssembly SIMD compatible browsers in order to keep the template sizes small. We generally aim to maintain compatibility with the oldest devices we can. But in this case the performance gains are too large to ignore and the chances of users have browser that are that far out of date is too small relative to the potential benefits. 
```

Reply:
> @clayjohn I'm already using "Starting from" in the last section.

I instead used "From here on out, the 4.5 release official templates will only..."